FanartHandler/FanartSelectedOther.cs

```diff
@@ -865,8 +902,25 @@ public void AddSelectedMoviePropertys()
                            Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesBanner) ? Utils.GetFanartTVFileName(strIMDBID, null, null, Utils.FanartTV.MoviesBanner) : string.Empty);
         Utils.SetProperty("movie.cd.selected", 
                            Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesCDArt) ? Utils.GetFanartTVFileName(strIMDBID, null, null, Utils.FanartTV.MoviesCDArt) : string.Empty);
+        FanartAvailable = FanartAvailable || 
+                                     Utils.AnimatedFileExists(strIMDBID, null, null, Utils.Animated.MoviesPoster) ||
+                                     Utils.AnimatedFileExists(strIMDBID, null, null, Utils.Animated.MoviesBackground) ||
+                                     Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesClearArt) ||
+                                     Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesClearLogo) ||
+                                     Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesBanner) ||
+                                     Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesCDArt);

```

Comment:
It would be nice to optimize this piece of code, but if you don’t want to, you can leave it like that.
An example of optimization:
```C#
...
var picFound = false;
picFound = Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesClearArt);
FanartAvailable = FanartAvailable || picFound;
Utils.SetProperty("movie.clearart.selected",
                  picFound ? Utils.GetFanartTVFileName(strIMDBID, null, null, Utils.FanartTV.MoviesClearArt) : string.Empty);

picFound = Utils.FanartTVFileExists(strIMDBID, null, null, Utils.FanartTV.MoviesClearLogo)
FanartAvailable = FanartAvailable || picFound;
Utils.SetProperty("movie.banner.selected",
                  picFound ? Utils.GetFanartTVFileName(strIMDBID, null, null, Utils.FanartTV.MoviesBanner) : string.Empty);
...
```

Reply:
> I'd rather just get the clearart done.

Ok, I’ll have time to optimize it later.